slick/src/sphinx/gettingstarted.rst

```diff
@@ -33,7 +33,8 @@ following to your build definition - ``build.sbt`` or ``project/Build.scala``:
 .. parsed-literal::
   libraryDependencies ++= Seq(
     "com.typesafe.slick" %% "slick" % "|release|",
-    "org.slf4j" % "slf4j-nop" % "1.6.4"
+    "org.slf4j" % "slf4j-nop" % "1.6.4",
+    "com.typesafe.slick" %% "slick-hikaricp" % "3.1.1"
```

Comment:
What I was thinking, newbies have fewer problems if they add hikaricp unnecessarily compared to when they don't add it when it is needed. So putting it there defensively is probably a good thing. Also, trying to remember if sphinx supports variables that inject the current version ... not sure